ajax.js

```diff
@@ -25,51 +30,103 @@ define('ajax', ['qwest'], function(qwest) {
 		};
 	}
 
+	function urlEncode(data) {
+		if (data) {
+			var encodedParams = [];
+			for (var k in data) {
+				encodedParams.push(encodeURIComponent(k) + '=' + encodeURIComponent(data[k]));
+			}
+			return encodedParams.join('&');
+		}
+		return null;
+	}
+
+	function getRequestObj() {
+		if (window.XMLHttpRequest) {
+			return new XMLHttpRequest();
+		} else if (window.ActiveXObject) {
+			return new ActiveXObject("Microsoft.XMLHTTP");
+		}
+	}
+
+	function isSuccess(status) {
+		return status >= 200 && status < 300;
+	}
+
+	function handleResponse(requestObj, callbacks) {
+		if (requestObj.readyState !== 4) {
+			return;
+		}
+
+		var responseType = requestObj.getResponseHeader("Content-Type") || "";
+		var responseContent = "";
+
+		if (responseType.indexOf("json") !== -1) {
```

Comment:
How about adding try/catch to catch parse exception? In this case if any error occurs at parsing we can call callback error like the others ajax lib.

@erikatakahara